__tests__/elements/mapchip/addfilterbutton.test.js

```diff
@@ -20,7 +20,7 @@ describe('Add filter button', () => {
         let component = new Component();
         let mc = new MapChip(component, mapchip_colors);
 
-        let dataFilterModel = new DataFilterModel(params.groups, params.chartConfiguration.filter, params.filter, params.primaryGroup, params.childData);
+        let dataFilterModel = new DataFilterModel(params.groups, params.chartConfiguration.filter, [], params.primaryGroup, params.childData);
```

Comment:
Are we are making this list empty because we don't need previous filters in the test?

Reply:
yes, and actually if we pass some previous filters the Jest would fail because it would need to render React components(snackbar).